js/ad_manager_vast.js

```diff
@@ -697,17 +851,27 @@ OO.Ads.manager(function(_, $) {
           indexInPod: adWrapper.ad.adPodIndex,
           skippable: false
         });
-      }
-      else {
+      } else {
         var streamUrl;
         if (adWrapper.ad && adWrapper.ad.streamUrl) {
           streamUrl = adWrapper.ad.streamUrl;
         }
         else if (adWrapper.streamUrl) {
           streamUrl = adWrapper.streamUrl;
+        } else {
+          streamUrl = '';
```

Comment:
What happens when streamUrl is the empty string? Will AMC get notified of an error?